src/symbol_mt5.py

```diff
@@ -57,8 +59,11 @@ def get_symbol_contract_size(self) -> float:
         return trade_contract_size
     
     def get_symbol_info_tick(self) -> dict:
-        symbol_info_tick = mt5.symbol_info_tick(self.symbol)._asdict()
-        return symbol_info_tick
+        symbol_info_tick = mt5.symbol_info_tick(self.symbol_name)
+        if symbol_info_tick == None:
```

Comment:
think we should use 'is' instead of '=='
https://sourcery.ai/blog/python-pandas-compare-to-none/#:~:text=PEP%208%20has%20a%20clear,could%20lead%20to%20unexpected%20results.